### ISSUE TITLE
Packaging: Add to homebrew

### ISSUE THREAD

**Issue by drnic** [tobimensch/termsql#3]
I'm vaguely hesitant from `sudo python setup.py install` install instructions. But for others, they might never get pass the pretty home page which doesn't mention how to install at all.

For OS X users, we like to guess how to install stuff -  `brew install termsql` or it doesn't exist :)

Perhaps look to https://github.com/jordansissel/fpm to create deb/yum packages too.

**Comment by tobimensch** [comment]
Thanks for your input.

In regards to packaging my next goal is to publish on pypi.

If someone wants to help with packaging for any specific platform patches are welcome.

**Comment by michaelmior** [comment]
Homebrew packages are quite easy to create. Unfortunately I no longer have a Mac available to actually test this, but there's great documentation [here](https://github.com/Homebrew/homebrew/blob/master/share/doc/homebrew/Python-for-Formula-Authors.md).

Any reason not to publish the current version of termsql on PyPI?

**Comment by tobimensch** [comment]
@michaelmior 

I don't have a Mac either, so I'm dependend on contributions for this platform.

There's no reason not to publish the current version on PyPI, a few months ago I started to setup the neccessary files and to register an account on PyPI, but I obviously didn't complete the process yet.

**Comment by aemreunal** [comment]
+1

**Comment by kaeff** [comment]
Hi @tobimensch , thanks for this awesome tool! As you asked for support with Homebrew packaging I took the liberty to propose a formula to Homebrew. I'm not experienced with writing brew formulas or packaging python projects, but I've made sure to follow the docs as best as I can. Happy for your feedback, thanks in advance!

From a Homebrew perspective, I think it would be super helpful if:

- There were tagged versions, so that the Formula can reference a tarball instead of cloning the repo and installing from master
- The path to the `man1` dir could be customized when calling `setup.py install`. Homebrew uses a dir called `prefix` to which it installs and then symlinks, so in order to install to the correct directory, as of now the formula needs to modify `setup.py` before installing. This felt a little hacky to me.

OR, ignoring the above: If there was a package on pypi, brew could just install that and be fine.

**Comment by kaeff** [comment]
As @woodrufw pointed out in Homebrew/homebrew-core#7801, Homebrew needs a version tag or full release for termsql. @tobimensch could you add one of those, please?

**Comment by tobimensch** [comment]
Yes, please be patient. I think I'll do it this week. Thanks for your contributions!

**Comment by tobimensch** [comment]
termsql is now on pypi, does this help with packaging for homebrew? Or is something else needed? I'm also tagging the current version as 1.0.
